packages/webstudio-sdk/.github/workflow/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Main workflow
```

Comment:
I think we want to automate publishing to npm for SDK package though

Reply:
Actually we will have to also publish every monorepo package SDK will depend on and since we want to be able to shift things easily around, we probably need to be publishing every package to npm